src/main/groovy/co/tala/performance/flo/FloRunna.groovy

```diff
@@ -78,5 +81,68 @@ class FloRunna<T> {
         results
     }
 
+    /**
+     * Executes a [WorkFlo] in parallel as a load test. Pass a [Closure<WorkFlo<T>>], which will be run repetitively,
+     * (based on the number of iterations specified) in parallel until the end of the test run. The threads,
+     * iterations, and rampup are defined in [FloRunnaSettings].
+     * @param workFloClosure
+     * @return
+     */
+    Map<String, FloExecutionResult<T>> executeIterations(
```

Comment:
This code seems copy paste of the other method. I think that it would be cleaner to still use the same original method. Setting the iterations could be optional. Ideally, we should use both the duration and the iterations. Whichever is reached first will then cause the test to finish. If iterations is not set, then we can set it to Int.MAX by default so that it would only exit on end of duration.

```
while (getElapsed() < duration || currentIteration <= iteration)
```

Reply:
One problem with `||`, is, if the tester does not set `iterations, then it will default to `Integer.MAX_VALUE`, which is 65K+ iterations. The duration would finish first and since it is a `||`, the test would continue until we have reached `Integer.MAX_VALUE`.

So we probably need to default iterations to `0` and if user wants to override, they can.